test/servinge2e/verify_networkpolicy_test.go

```diff
@@ -44,9 +46,17 @@ func TestNetworkPolicy(t *testing.T) {
 	}
 	defer caCtx.Clients.Kube.NetworkingV1().NetworkPolicies(testNamespace3).Delete(policyNameDeny, &metav1.DeleteOptions{})
 
-	_, err = http.Get(ksvc.Status.URL.String())
-	if err == nil {
-		t.Fatalf("Netowrk policy did not block the request to %s", ksvc.Status.URL.String())
+	// Poll until network policy became active. It takes a few seconds.
+	err = wait.PollImmediate(test.Interval, 1*time.Minute, func() (bool, error) {
+		_, inErr := http.Get(ksvc.Status.URL.String())
+		if inErr == nil {
+			t.Logf("Netowrk policy did not block the request to %s", ksvc.Status.URL.String())
```

Comment:
```suggestion
			t.Logf("Network policy did not block the request to %s", ksvc.Status.URL.String())
```

everywhere please :D